### ISSUE TITLE
Fail to generate blog local preview

### ISSUE THREAD

**Issue by brunoeducsantos** [fastai/fastpages#474]
Hi ,
I was trying to generate local preview of my blog. 
The steps I performed were:
1. `sudo make build`
2. `sudo make server`
My OS is ubuntu 16.04.
Getting the following error:

```
watcher_1    | converting: _notebooks/2020-02-20-test.ipynb
jekyll_1     | Configuration file: /data/_config.yml
jekyll_1     | /usr/local/lib/ruby/2.7.0/fileutils.rb:250:in `mkdir': Permission denied @ dir_s_mkdir - /data/.jekyll-cache (Errno::EACCES)
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:250:in `fu_mkdir'
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:228:in `block (2 levels) in mkdir_p'
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:226:in `reverse_each'
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:226:in `block in mkdir_p'
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:211:in `each'
jekyll_1     | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:211:in `mkdir_p'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/cache.rb:184:in `dump'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/cache.rb:101:in `[]='
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/cache.rb:45:in `clear_if_config_changed'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/site.rb:118:in `reset'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/site.rb:35:in `initialize'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:30:in `new'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:30:in `process'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
jekyll_1     | 	from /usr/gem/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
jekyll_1     | 	from /usr/gem/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
jekyll_1     | 	from /usr/gem/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
jekyll_1     | 	from /usr/gem/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
jekyll_1     | 	from /usr/gem/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
jekyll_1     | 	from /usr/gem/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>'
jekyll_1     | 	from /usr/local/bundle/bin/jekyll:29:in `load'
jekyll_1     | 	from /usr/local/bundle/bin/jekyll:29:in `<main>'

```

Any help would be appreciated.
Regards,
Bruno

**Comment by hamelsmu** [comment]
Can you change permissions on your local folder(s)?  Also you what version of fastpages are you using?  Are you using the latest version?

**Comment by hamelsmu** [comment]
I'm not able to reproduce this error, which means perhaps something funny is going on with file permissions on your local computer or perhaps how Docker is setup?  If you try granting permissions to all files recursively in the directory, it should work.  I was able to clone your repo and run it locally without issue

![image](https://user-images.githubusercontent.com/1483922/103103709-cf1cfa00-45d7-11eb-9961-3c9bf8c2bab1.png)

Closing this issue but feel free to continue the conversation.
